core/config/docs/core.toml

```diff
@@ -472,6 +472,9 @@ MaxBinarySize = '20.00mb' # Default
 MaxEncryptedSecretsSize = '26.40kb' # Default
 # MaxConfigSize is the maximum size of a config that can be fetched from the given config url.
 MaxConfigSize = '50.00kb' # Default
+# SyncStrategy is the strategy that will be used to bring the node up to date with the latest Workflow Registry contract state.
+# Options are: event which watches for contract events or reconciliation which diffs workflow metadata state.
+SyncStrategy = 'reconciliation' # Default
```

Comment:
This shouldn't be the default since we'll lose the benefits of the flag as the release mechanism